### DISCUSSION TITLE
Question on forced refresh configuration

### DISCUSSION THREAD

**Discussion by creativethings** [Hyundai-Kia-Connect/kia_uvo#716] [General]
Hi all,
I just started to use the integration and so far I am very impressed. Many thanks for all the work on this.
These I think I understand:
- Force Request Interval: sets the timeout in minutes for forcing the app to get fresh data from the KIA server 
- No Force Refresh Start Hour: time after which there will be no forced refresh
- No Force Refresh Finish Hour: time after which there may be a forced refresh

So lets assume the "No Force Finish Hour" is set to 06.00 AM and my "Force Request Interval" is set to 2 hours, will the first forced refresh be at 06:00 or at 08:00?
thx!
Niels

**Comment by cdnninja** [comment]
Force refresh is calling the car not the cached data on the servers.  The only occurs if the cached data is older than the set interval.  Use sparingly since it uses 12v on the car.
